integrationtests/logsCrossShard_test.go

```diff
@@ -31,15 +31,18 @@ func TestIndexLogSourceShardAndAfterDestinationAndAgainSource(t *testing.T) {
 	}
 	body := &dataBlock.Body{}
 
+	address1 := "erd1ju8pkvg57cwdmjsjx58jlmnuf4l9yspstrhr9tgsrt98n9edpm2qtlgy99"
+	address2 := "erd1w7jyzuj6cv4ngw8luhlkakatjpmjh3ql95lmxphd3vssc4vpymks6k5th7"
+
 	// INDEX ON SOURCE
 	pool := &outport.Pool{
 		Logs: []*coreData.LogData{
 			{
 				LogHandler: &transaction.Log{
-					Address: []byte("addr-1"),
+					Address: decodeAddress(address1),
 					Events: []*transaction.Event{
 						{
-							Address:    []byte("addr"),
+							Address:    decodeAddress(address1),
```

Comment:
Why not used different addresses as before?

Initially, you had `[]byte("addr-1")` & `[]byte("addr")`, but now you replaced them both with `address1`

Reply:
Applies for addresses below in this file as well

Reply:
these addresses do not matter; I also changed them also in the file with the expected result.